src/stories/components/Tag.stories.tsx

```diff
@@ -29,19 +41,27 @@ export default {
   },
 } as Meta<typeof Tag>;
 
-const Template: StoryFn<typeof Tag> = (args) => <Tag {...args} />;
+const Template: StoryFn<CustomTagProps> = (args) => (
+  <Tag {...args}>
+    <TagLabel>{args.children}</TagLabel>
+    {args.closable && <TagCloseButton />}
```

Comment:
```suggestion
    <TagLabel>{args.children}</TagLabel>
    {args.closable && <TagCloseButton />}
```

Are the `TagLabel` and `TagCloseButton` components exported by Boemly to be able to use it in the `app` or wherever Boemly is used? Otherwise this might not be possible.